org/eclipse/persistence/org.eclipse.persistence.antlr/maven-metadata.xml

```diff
@@ -26,7 +26,8 @@
       <version>2.7.4.payara-p2</version>
       <version>2.6.5.payara-p4</version>
       <version>2.6.8.payara-p1</version>
+      <version>2.7.6.payara-p1</version>
```

Comment:
The same as in [comment](https://github.com/payara/Payara_PatchedProjects/pull/303/files?file-filters%5B%5D=.md5&file-filters%5B%5D=.sha1&file-filters%5B%5D=.xml#r416516593) above.